### ISSUE TITLE
Request of IDs for MW meanings (additional to L for entries)

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/MWS#19]
Let me explain https://github.com/sanskrit-lexicon/MWS/issues/7 "24) Can we get a unique ID for every meaning in MW as well?" in more details. Otherwise it could get lost even for the sake of discussion.

I've been working with MW for ages and have tried a lot to understand what we should do with it (just today stumbled upon http://www.dialog-21.ru/digests/dialog2006/materials/html/Gasuns.htm) in the future.
Let me tell you my story, why I need to have an ID for each "translation" as well.
1) I want to translate ayurvedic literature to Russian. Possible victims:
- http://ayurvedam.ru/nadipariksa-english-wordbyword/
- http://ayurvedam.ru/susruta-samhita/
- http://ayurvedam.ru/caraka-samhita/
- http://ayurvedam.ru/astanga-hrdaya-samhita/

I've heard the the 2nd German translation took 6.5 hears to make it. And it was not even a word by word translation, so I'm not sure if it's a good idea, but my ayurvedic doctor wants to translate AHS. That means I have to translate it myself before he does it so I can help him in some way. 
2) But before it becomes Russian, I need to make an English word by word translation. The good news is that Oliver has already partly solved the task in 2012 at his DSC. He has assigned an ID to every word (not meaning) and I'm not aware weather the AHS has any missasignments, for that I'll need additional help from an Indian ayurvedic researcher. The bad news - at some point and he does not remember why Oliver sorted the meanings inside every dictionary entry alphabetically (see http://samskrtam.ru/dsc-bugs/). That means that the frequent, wanted meanings starting with `z` will never be there, because the logical order of the entries is broken. Only because of that a huge amount of work has to be redone now.
A single example. अर्थhas first (=main) meaning `aim`. But in Oliver's "new order" it's fifth and as I take only the first 3 meanings of every entry, it is left out in my automatic word by word translation file.
3) If every translation will have it's own ID similar to the L numbers, than I will be able to start develop a system of improving the quallity of the word by word translations. First of all I will include:
- translations marked with Med. tag
- translations marked with Caraka and other medical authorities
- translations in the beginning of the list, those that come first

After that I would make a voting system. An ayurvedic doctor reads the text and clicks on the most appropriate word out from the 3-5 word list. We gather statistics for these words http://kjc-fs-cluster.kjc.uni-heidelberg.de/dcs/index.php?contents=textdictionary&IDText=10 which meaning they preffer. After that maybe even an auto correction mode can be made. We "vote" for words in one chapter and the quallity score for different chapters might improve as well. We would need to map which `L` nubmers match witch `einzelwort&IDWord` at Oliver (sample http://kjc-fs-cluster.kjc.uni-heidelberg.de/dcs/index.php?contents=einzelwort&IDWord=104 = L=224 and `without a why or a wherefore, accidentally, suddenly.` is split by `,`). There are 72 000 words used at Oliver's version of MW.
4) After that I will need to try to match Russian translations of MW at http://www.yukta.org/download.php?lang=rus and verify them at my Parallel Sanskrit Corpus, like http://samskrtam.ru/parallel-mahabharata/
I do not know if I should try to make a macro to work inside Word, Adobe Acrobat or some web UI. That is not that important know. What matters is that every meaning should have it's ID. Does it seams reasonable and possible, @funderburkjim ? @drdhaval2785 , is it possible to understand what I speak about?

**Comment by drdhaval2785** [comment]
@gasyoun, I am not sure whether I understand you here. Can you rephrase what you want done?

**Comment by Andhrabharati** [comment]
@funderburkjim / @gasyoun,

Is this issue closable now?
